components/shooterLogic.py

```diff
@@ -1,7 +1,7 @@
 from robotMap import XboxMap
+import logging as log
```

Comment:
Needed?

Reply:
Yes, we use it multiple times throughout the file